test/common.py

```diff
@@ -2306,7 +2316,7 @@ def do_GET(self):
           # tell the browser to load the test
           self.wfile.write(b'COMMAND:' + url.encode('utf-8'))
           # move us to the right place to serve the files for the new test
-          os.chdir(dir)
+          base_path = dir
```

Comment:
Can we land the change to stop using `chdir` separately perhaps?   It might be nice to review that part in isolation.